packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -51,6 +51,9 @@ const ALERT_SUPPRESSION_START = `${ALERT_SUPPRESSION_META}.start` as const;
 const ALERT_SUPPRESSION_END = `${ALERT_SUPPRESSION_META}.end` as const;
 const ALERT_SUPPRESSION_DOCS_COUNT = `${ALERT_SUPPRESSION_META}.docs_count` as const;
 
+// Fields pertaining to the cases associated with the alert
+const ALERT_CASES_IDS = `${ALERT_NAMESPACE}.cases_ids` as const;
```

Comment:
Sorry to be annoying but we did agree on `case_ids`

Reply:
https://docs.google.com/document/d/1C283LLAMniXS-CXT75gpLu_c1ltWtMcIIw1PeAUxbS4/edit?pli=1

Reply:
Done!